client/flutter/lib/pages/latest_numbers.dart

```diff
@@ -20,118 +20,167 @@ const name = TextStyle(
-                return SliverList(
-                    delegate: SliverChildListDelegate([
-                  StatCard(
-                      background: LatestNumbersGraph(
-                        timeseries:
-                            hasGlobalStats ? globalStats['timeseries'] : null,
-                        timeseriesKey: 'totalCases',
-                      ),
-                      title: Text(
-                        S.of(context).latestNumbersPageGlobalCasesTitle,
-                        style: name,
-                      ),
-                      content: Text(
-                        hasGlobalStats && globalStats['cases'] != null
-                            ? numFmt.format(globalStats['cases'])
-                            : '-',
-                        softWrap: true,
-                        style: hasGlobalStats && globalStats['cases'] != null
-                            ? number
-                            : loadingStyle,
-                        textAlign: TextAlign.left,
-                      )),
-                  StatCard(
-                    background: LatestNumbersGraph(
-                      timeseries:
-                          hasGlobalStats ? globalStats['timeseries'] : null,
-                      timeseriesKey: 'totalDeaths',
-                    ),
-                    title: Text(S.of(context).latestNumbersPageGlobalDeaths,
-                        style: name),
-                    content: Text(
-                      hasGlobalStats && globalStats['deaths'] != null
-                          ? numFmt.format(globalStats['deaths'])
-                          : '-',
-                      softWrap: true,
-                      style: hasGlobalStats && globalStats['deaths'] != null
-                          ? number
-                          : loadingStyle,
-                      textAlign: TextAlign.left,
-                    ),
-                  ),
-                  Container(
-                    height: 25,
-                  ),
-                  Text(
-                    snapshot.hasData
-                        ? S.of(context).latestNumbersPageLastUpdated(lastUpd)
-                        : S.of(context).latestNumbersPageUpdating,
-                    style: TextStyle(color: Color(0xff26354E)),
-                    textAlign: TextAlign.center,
-                  ),
-                  Container(
-                    height: 10,
+          CupertinoSliverRefreshControl(
+            refreshIndicatorExtent: 100,
+            refreshTriggerPullDistance: 100,
+            builder:
+                (context, RefreshIndicatorMode refreshIndicatorMode, a, b, c) {
+              double topPadding = a > 25 ? a : 25.0;
+              var loadingIndicator = Padding(
+                padding: EdgeInsets.only(top: topPadding - 25),
+                child: CupertinoActivityIndicator(),
+              );
+              switch (refreshIndicatorMode.index) {
```

Comment:
It would be clearer to switch on the enum itself here rather than the integer value...